backend/main.py

```diff
@@ -113,6 +113,36 @@ def new_game():
   return jsonify(firebase.put('/games', game, put_data))
 
 
+@app.route('/joingame', methods=['POST'])
```

Comment:
That was based on '/creategame' which wasn't camel, but I agree that it should be changed to match the majority of the routes.